vowpalwabbit/global_data.h

```diff
@@ -168,18 +168,18 @@ struct shared_data
   float second_observed_label;
 
   // Column width, precision constants:
-  static const int col_avg_loss = 8;
-  static const int prec_avg_loss = 6;
-  static const int col_since_last = 8;
-  static const int prec_since_last = 6;
-  static const int col_example_counter = 12;
-  static const int col_example_weight = col_example_counter + 2;
-  static const int prec_example_weight = 1;
-  static const int col_current_label = 8;
-  static const int prec_current_label = 4;
-  static const int col_current_predict = 8;
-  static const int prec_current_predict = 4;
-  static const int col_current_features = 8;
+  static constexpr int col_avg_loss = 8;
```

Comment:
[Minor][Nit] There are inconsistencies between `static constexpr` and `constexpr static`. Is there a reason for that?

Reply:
No reason, I just replaced const with cosntexpr so they were ordered the other way around before and I didn't notice